microservices/device-simulation-dotnet/scripts/docker/build.cmd

```diff
@@ -2,10 +2,14 @@
 
 @ECHO off & setlocal enableextensions enabledelayedexpansion
```

Comment:
This file looks very similar to `microservices/api-gateway/build.cmd`, but that file did not include what you have here for line 3.  Was that intentional and if so, can you provide an inline comment for one or the other?

Reply:
Unknown. These are legacy build scripts from the previous/original microservice repositories. I don't want to speculate as to why one has these and another script does not. Plus, I don't think that's a good use of time.